services/core-api/app/api/projects/project/resources/project.py

```diff
@@ -276,11 +285,36 @@ def _apply_filters_and_pagination(self, args):
                 ]
                 conditions.append({'or': update_timestamp})
 
+        if args['project_lead_name'] is not None:
+            lead_items = args['project_lead_name'].split()
+            project_lead_conditions=[]
+            if len(lead_items) == 1:
+                project_lead_conditions = [
+                    self._build_filter('Party','first_name','ilike','%{}%'.format(lead_items[0])),
+                    self._build_filter('Party','party_name','ilike','%{}%'.format(lead_items[0]))
+                ]
+            elif len(lead_items) > 1:
+                project_lead_conditions = [
+                    self._build_filter('Party','first_name','ilike','%{}%'.format(lead_items[:-1])),
+                    self._build_filter('Party','party_name','ilike','%{}%'.format(lead_items[-1]))
+                ]
+            else:
+                project_lead_conditions = []
+
+            if len(project_lead_conditions) == 2:
+                conditions.append({'or': project_lead_conditions})
+
         projects_query = apply_filters(query, conditions)
 
         # Apply sorting
         if sort_model and args['sort_field'] and args['sort_dir']:
-            sort_criteria = [{'model': sort_model, 'field': args['sort_field'], 'direction': args['sort_dir']}]
+            sort_criteria=[]
+            if sort_model == 'Party':
```

Comment:
I notice when using the column sorter that all entries which do not have a project_lead set do not get returned in the sorted result, is that to be expected?

Reply:
Fixed.